modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,524 @@
+= Add a BYOC Customer-Managed VPC on GCP
+:description: Use the Redpanda UI to create a customer-managed VPC peering connection for a BYOC cluster.
+:page-cloud: true
+
+You can connect a Redpanda Cloud BYOC cluster with your existing virtual private cloud (VPC). Using your own VPC gives you control over your security policies and resources, including service accounts, IAM roles, subnetworks, firewall rules, and storage buckets. 
+
+== Prerequisites
+
+* A GCP account
+* A running BYOC cluster in GCP. See xref:./create-byoc-cluster-gcp.adoc[Create a BYOC Cluster on GCP]. Using your own VPC, you select the *Customer Managed* network connection type.
```

Comment:
The BYOC cluster will not yet be running, this is an alternative workflow for creating a BYOC cluster. I'd be happy to do a demo if that's helpful.

Reply:
yes, please! I'll get something on your calendar!